src/model.ts

```diff
@@ -342,6 +358,7 @@ export class JSORMBase {
     JsonapiResourceIdentifier[]
   > = {}
   @nonenumerable private _attributes: ModelRecord<this>
+  @nonenumerable private _attributeOverrides: ModelRecord<this>
```

Comment:
Might be helpful to put a comment here for the future to describe how `attributes`, `attributeOverrides`, and `originalAttributes` all relate to one another.